src/main/java/gov/nih/ncats/molwitch/spi/ChemicalImplFactory.java

```diff
@@ -219,4 +220,9 @@ public interface ChemicalImplFactory {
      * @since 0.5.2
      */
     boolean isFormatAgnostic();
+
+	static void applyParameters(Map<String, Object> params) {
+		System.out.println("ChemicalImplFactory default applyParameters");
```

Comment:
I think this shouldn't be a static method but a instance method of the interface with a no-op default. Otherwise there's no inheritance.

Reply:
So if should be:

```
 default void applyParameters(Map<String, Object> params) {
```